api/src/main/java/keywhiz/api/automation/v2/SecretDetailResponseV2.java

```diff
@@ -64,9 +68,21 @@ public Builder secret(Secret secret) {
           .createdAtSeconds(secret.getCreatedAt().toEpochSecond())
           .createdBy(secret.getCreatedBy())
           .type(secret.getType().orElse(null))
+          .expiry(secret.getExpiry())
           .metadata(secret.getMetadata());
     }
 
+    public Builder secretContent(SecretContent secretContent) {
```

Comment:
Hmm.  I named it for parallelism with the methods series(SecretSeries) and secret(Secret), but silently discarding the encrypted content could be confusing (the other two methods fill the content() field).  For now, I'll just leave the field out.